ui/pages/keyring-snaps/add-snap-account-modal/add-snap-account-modal.tsx

```diff
@@ -4,11 +4,11 @@ import {
   Box,
   Button,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Text,
 } from '../../../components/component-library';
+import { ModalContent } from '../../../components/component-library/modal-content/deprecated';
+import { ModalHeader } from '../../../components/component-library/modal-header/deprecated';
 import {
   AlignItems,
   Display,
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 5 03 02 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/4183b587-5c7a-4f4b-90f1-0663a14353b0">

### After
I did not fix the image it seems to be an issue with hosting via github pages
<img width="1512" alt="Screenshot 2023-12-10 at 5 03 15 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/82f57a6b-c370-424c-9601-039b0796b433">